src/v2/Apps/Home/Components/HomeFeaturedEventsRail.tsx

```diff
@@ -30,47 +31,78 @@ const HomeFeaturedEventsRail: React.FC<HomeFeaturedEventsRailProps> = ({
   if (events.length === 0) return null
 
   return (
-    <GridColumns>
-      {events.map((event, i) => {
-        const image = event.image?.cropped
+    <>
+      <Text variant="lg" mb={4}>
+        Featured
+      </Text>
+
+      <GridColumns gridRowGap={2}>
+        {events.map((event, i) => {
+          const image = event.image
+
+          return (
+            <Column key={event.internalID ?? i} span={3}>
+              <RouterLink
+                to={event.href ?? ""}
+                style={{ display: "block", textDecoration: "none" }}
+              >
+                <Media lessThan="sm">
+                  <Flex>
+                    {image?.small ? (
+                      <Image
+                        src={image.small.src}
+                        srcSet={image.small.srcSet}
+                        width={image.small.width}
+                        height={image.small.height}
+                        alt=""
+                        lazyLoad
+                      />
+                    ) : (
+                      <Box bg="black10" width="100%" height="100%" />
+                    )}
 
-        return (
-          <Column key={event.internalID ?? i} span={3}>
-            <RouterLink
-              to={event.href ?? ""}
-              style={{ display: "block", textDecoration: "none" }}
-            >
-              <Flex alignItems="center">
-                <Box flexShrink={0}>
-                  {image ? (
-                    <Image
-                      src={image.src}
-                      srcSet={image.srcSet}
-                      width={95}
-                      height={63}
-                      alt=""
-                      lazyLoad
-                    />
-                  ) : (
-                    <Box bg="black10" width={95} height={63} />
-                  )}
-                </Box>
+                    <Box ml={2}>
+                      <Text variant="xs" textTransform="uppercase">
+                        {event.title}
+                      </Text>
 
-                <Spacer ml={2} />
+                      <Text variant="lg">{event.subtitle}</Text>
+                    </Box>
+                  </Flex>
+                </Media>
 
-                <Flex flexDirection="column" justifyContent="center">
-                  <Text variant="xs" color="black60">
+                <Media greaterThanOrEqual="sm">
+                  <ResponsiveBox
+                    aspectWidth={3}
+                    aspectHeight={2}
```

Comment:
What do these numbers do?

Reply:
They set the aspect ratio. So this becomes a responsive 3:2 box. In this case I could also use the width/height coming off the crop